circle.yml

```diff
@@ -17,6 +17,21 @@ jobs:
         command: |
           codespell --quiet-level=4 --ignore-words=./.codespell-whitelist
 
+  bumpversion:
+    docker:
+    docker:
+    - image: ethereum/cpp-build-env:10
+    steps:
+    - checkout
+    - run:
+        name: "Check bumpversion"
+        command: |
+          pip3 install bumpversion
```

Comment:
Don't you want `pip3 install --user`?

Reply:
Dunno, can try.

Reply:
That uses the same path, still needs the `export PATH`.